### DISCUSSION TITLE
Welcome to hacs-hubitat Discussions!

### DISCUSSION THREAD

**Discussion by jason0x43** [jason0x43/hacs-hubitat#93] [General]
## 👋 Welcome!

This is a place to connect with other users of hacs-hubitat. Please, feel free to:

  * Ask questions
  * Share ideas
  * Engage with other community members

**Comment by ewalker101** [comment]
Hi jason0x43, This is great work. Got my Hubitat C-7  today and have already started integrating Zigbee devices into it. I found your integration was very easy to use once I read your documentation. Thanks again for this awesome work. Now the laborious work starts for me as Ill need to move all of my Zigbee devices from my Sonoff Tasmota hub to Hubitat and reestablish my automatons.
